content/concepts/basics/03-content-relations.md

```diff
@@ -4,17 +4,68 @@ metaTitle: "sensenet - Content Relations"
 metaDescription: "sensenet Content Relations"
 ---
 
-# Content relations
+In sensenet the relation between contents is as much important as storing them. Because of the hierarchical tree structure of the content repository, it's important to mention parent and child relations between contents allowing users to take advantage of inheritance. Beside that, sensenet also gives the possibility to create other type of relations between contents. Such relation could be in a form of reference field (link to another content), binary field (for storing binary data on a content) or attachment (by combining reference and binary field).
+
 # Content repository tree
-# Parent
-# Child
-# Links
-https://wiki.sensenet.com/Reference_Field.html (nyilván innen értelemszerűen csak maga a tárolási/működési elv, ami kell)
-# Multiple links
-https://wiki.sensenet.com/Reference_Field.html (nyilván innen értelemszerűen csak maga a tárolási/működési elv, ami kell)
+
+sensenet content repository is basically a tree structure of the various stored content (e.g. folders, documents, user, tasks, etc.).
+This structure lets you organize, group your content easily by creating separate subtrees. It's not only important because of the possibility to configure content accessibility for different types of users on different parts of the tree, but also because of the inheritance of several things like content types' underlying business logic, metadata field configurations and settings.
+
+# Parent-child relation
```

Comment:
vajon le kell írni, h mit is jelent konkrétan a parent-child viszony, mikor beszélünk róla?